src/Recruit.Vacancies.Client/Domain/Enums/VacancyStatus.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Esfa.Recruit.Storage.Client.Domain.Enum
```

Comment:
I don't think we need a separate namespace for enum.

Reply:
Agree. Will fix